data/CFOUR/basicCFOUR2.1/README.md

```diff
@@ -0,0 +1,5 @@
+# CFOUR notes
```

Comment:
:+1: 

Maybe put this in `data/CFOUR` though? It will probably always be relevant, regardless of the version.

Reply:
Agreed, this has been moved up to the CFOUR folder.